gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -486,7 +488,9 @@ def add_arguments(self, parser):
     parser.add_argument(
         '--representative_header_file',
         help=('If provided, meta-information from the provided file will be '
-              'added into the output_file.'))
+              'added into the output_file. Otherwise, the meta-information is '
+              'inferred from the BigQuery schema. To best recover the VCF, it'
```

Comment:
nit: consider rephrasing as `It is recommended to provide this file to specify the most accurate and complete meta-information.`